pkg/cluster/controllers/storage/minio/apis/minio.min.io/v2/types.go

```diff
@@ -0,0 +1,926 @@
+package v2
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:defaulter-gen=true
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Namespaced,shortName=tenant,singular=tenant
+// +kubebuilder:printcolumn:name="State",type="string",JSONPath=".status.currentState"
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:storageversion
+
```

Comment:
These comments need to be removed and add `// +kubebuilder:skip` in this file.

Reply:
Why `// +kubebuilder:skip`? Having crd matching these structure is not needed? Then how to create/update these resources?